MyCryptoHQ/MyCrypto
# Fixes arbitrary value for gasPrice

Closes #2304

### Description

If you try to send a transaction to a contract, the initial gas price is always 20 Gwei. This is maybe right for Ethereum but not for other chains.

### Changes

* `mapStateToProps` in `TXMetaDataPanel`, getting now gas price from settings if network isn't a custom one.

### Steps to Test

1. Go to "Interact with contracts"
2. Select any network that its initial gas price is not 20 Gwei, i.e. `RSK`
3. Prepare a transaction to a contract
4. Check that Gas Price (gwei) field has a value of `0.06`
